### DISCUSSION TITLE
What is replacement for deprecated InfinispanConfigurationCustomizer?

### DISCUSSION THREAD

**Discussion by nikolas-charalambidis** [infinispan/infinispan#12138] [Q&A] [answered]
I am using `infinispan-spring-boot3-starter-embedded-14.0.17.Final` and its `InfinispanConfigurationCustomizer` that is marked as `@Deprecated` however both its JavaDoc and the documentation for the [DEV version 15.0.X](https://infinispan.org/docs/15.0.x/titles/spring_boot/starter.html#embedded-custom-beans_embedded) fail to offer a replacement.

How to replace `InfinispanConfigurationCustomizer` properly?
Thanks a lot.

**Comment by karesti** [comment, marked as answer]
@Nikolas-Charalambidis Hi the `InfinispanCacheConfigurer` to configure your caches

```java
 @Bean
   public InfinispanCacheConfigurer cacheConfigurer() {
      return manager -> {
         final org.infinispan.configuration.cache.Configuration ispnConfig = new ConfigurationBuilder()
               .clustering().cacheMode(CacheMode.DIST_SYNC)
               .encoding().mediaType(MediaType.APPLICATION_PROTOSTREAM_TYPE)
               .statistics().disable()
               .build();


         manager.defineConfiguration("sessions", ispnConfig);
      };
   }
```

**Reply by nikolas-charalambidis** [reply]
It looks good. Thanks.
Is there an issue related to the documentation? The docs for 15.0.X are quite empty.

**Reply by karesti** [reply]
Supporting Spring Boot 2 and 3 was already done in Infinispan 14. In 15 we dropped Spring Boot 2
